### ISSUE TITLE
Add the ability to run a query without any scoping

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#2903]
Useful for administrators and query builder stats

**Comment by maxxxxxdlp** [comment]
Some difficulties:
What if user does not have access to some collection?
Then query would have to essentially be scoped to all but one collection

This issue is actually blocking #2676 - see https://github.com/specify/specify7/pull/2676#issuecomment-1399264791 for more details

**Comment by grantfitzsimmons** [comment]
>  These types of searches are not possible in the Specify 6.0 release, but Specify is designed and we are committed to enabling institutions to perform cross-collection searches, and we will include adding that capability in a future release

This is literally something we promised to release when Specify 6.0 was announced in our documentation in April 2009. ([Link](https://discourse.specifysoftware.org/t/specifys-organizational-model-and-user-accounts-for-multiple-collections/656))

This could be enabled **only** for institution admin users.

**Comment by maxxxxxdlp** [comment]
From @grantfitzsimmons:

> There has to be an issue for this?! I can't find it after searching extensively. Need to focus on this meeting.
>
> **Requested By:** NMBE

**Comment by grantfitzsimmons** [comment]
Ugh– new title.

**Comment by grantfitzsimmons** [comment]
The University of Michigan has inquired about this feature.

My two cents:
Disciplines share the same schema, but different pick lists. We could, at the very least, introduce cross-discipline queries first.

**Comment by grantfitzsimmons** [comment]
> It’s too bad that cross collection searches are not an option in Specify at this point. We were hoping to take advantage of taxonomically focused collections to better implement schemas and form views that best capture different taxonomic groups. If searching across collections isn’t going to be possible now or in the near future, we need to consider other options carefully.

**Comment by melton-jason** [comment]
Yes, we definitely need to improve how different scoping levels interact with each other! 
Decouple our components from the scoping, and allow the user to define which scope they are interacting with. (Easier said then done). 

 It seems this is becoming more and more of a priority. 
More so than Batch Edit? I would imagine the two would require a similar investment in terms of time. 
If we develop the two features concurrently, we have to be very cautious we don't repeat xml-editing, agent-merging, and business rules!

**Comment by maxxxxxdlp** [comment]
Not scoping queries can be done relatively easily in hacky ways and with limitations (I.e, only allow it for admins and don't handle pick lists)
Batch editing even with hacks (reusing a lot of query builder and workbench code) would still require quite a bit of work

**Comment by maxxxxxdlp** [comment]
Not sure how you would go about implementing this, but you might want to keep https://github.com/specify/specify7/issues/2912#issuecomment-1406828873 in mind just in case it's related

**Comment by grantfitzsimmons** [comment]
CSIRO mentioned needing this ability again.

**Comment by grantfitzsimmons** [comment]
Anu at CSIRO mentioned this again today

**Comment by grantfitzsimmons** [comment]
See how this can be accomplished in Specify 6: https://discourse.specifysoftware.org/t/allow-queries-to-search-all-collections-in-a-database/768

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/search-across-collections/2235/2
